easybuild/easyconfigs/s/scenicplus/scenicplus-1.0.0-foss-2022a.eb

```diff
@@ -0,0 +1,353 @@
+easyblock = 'PythonBundle'
+
+name = 'scenicplus'
+version = '1.0.0'
+
+homepage = 'https://github.com/aertslab/scenicplus'
+description = """SCENIC+ is a python package to build gene regulatory networks (GRNs)
+using combined or separate single-cell gene expression (scRNA-seq) and single-cell
+chromatin accessibility (scATAC-seq) data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
+    ('Ray-project', '2.2.0'),
+    # ('attr', '2.5.1'),
+    ('typing-extensions', '4.3.0'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-learn', '1.1.2'),
+    ('networkx', '2.8.4'),
+    ('Seaborn', '0.12.1'),
+    ('pyBigWig', '0.3.18'),
+    ('plotly.py', '5.12.0'),
+    ('MACS2', '2.2.9.1'),
+    ('lxml', '4.9.1'),
+    ('Kaleido', '0.2.1'),
+    ('IPython', '8.5.0'),
+    ('scanpy', '1.9.1'),
+    ('Cython', '0.29.33'),
+    ('statsmodels', '0.13.1'),
+    ('h5py', '3.7.0'),
+    ('adjustText', '0.7.3'),
+    ('umap-learn', '0.5.3'),
+    ('ipympl', '0.9.3'),
+    ('leidenalg', '0.9.1'),
+    ('loompy', '3.0.7'),
+    ('numba', '0.56.4'),
+    ('polars', '0.15.6'),
+    ('pybedtools', '0.9.0'),
+    ('BeautifulSoup', '4.10.0'),
+    ('Sphinx-RTD-Theme', '1.3.0'),
+    ('Rust', '1.65.0'),  # required (newer version) for setuptools-rust, which is needed for gseapy
```

Comment:
`Rust` should probably only be a build dependency here, it's a pain to have this as runtime dependency...